chips/lowrisc/src/spi_host.rs

```diff
@@ -361,15 +361,14 @@ impl<'a> SpiHost<'a> {
     fn start_transceive(&self) {
         let regs = self.registers;
         //8-bits that describe command transfer len (cannot exceed 255)
-        let num_transfer_bytes: u32;
         //TXQD holds number of 32bit words
         let txfifo_num_bytes = regs.status.read(status::TXQD) * 4;
 
-        if txfifo_num_bytes > u8::MAX as u32 {
-            num_transfer_bytes = u8::MAX as u32;
+        let num_transfer_bytes: u32 = if txfifo_num_bytes > u8::MAX as u32 {
+            u8::MAX as u32
         } else {
-            num_transfer_bytes = txfifo_num_bytes;
-        }
+            txfifo_num_bytes
+        };
```

Comment:
The comment for this variable needs to move too

Reply:
Moved.